src/components/views/read/cvForm/Education.tsx

```diff
@@ -116,12 +158,18 @@ const Education: FC<Props> = ({
     });
   };
   const handlePrev = (): void => {
-    onCvValuesChange(educationCards);
+    const educationWithoutPresent = educationCards.map(
+      ({ present, ...rest }) => rest,
+    );
+    onCvValuesChange(educationWithoutPresent);
```

Comment:
I don't think you need this anymore, as all the values are coming form the appDataArray. If they are not in there, they are not saved so there is no point.